client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-definitions/plan/index.tsx

```diff
@@ -97,7 +100,35 @@ const getPlanCompletedTask: TaskAction = ( task, flow, context ) => {
 	};
 };
 
+//TODO: Move the updateLaunchpadSettings to be a hoook and use queryclient to invalidate the hook.
```

Comment:
Does this comment still apply or should it be removed?

Reply:
I am adding this comment to register future improvements.
Query client is an internal implementation detail related to hook that uses react-query. 
So when we use it outside the correct context is similar to use an database connection outside the database layer.

Reply:
It might be worth creating a Github issue for it as well.